### DISCUSSION TITLE
Media player attributes are missing stream_title:

### DISCUSSION THREAD

**Discussion by sp0ngeboobs** [music-assistant/support#2156] [Q&A]
after b139, the "stream_title:" attribute, which contained the name of the track when I was listening to radio streams, disappeared in the attributes of media players.

**Comment by OzGav** [comment]
Can you see it in any other app? Maybe the station stopped sending it?

**Reply by sp0ngeboobs** [reply]
HA core update came out today, stream_title went back after update. amazingly.

**Comment by sp0ngeboobs** [comment]
when I tested, changed a couple of radio stations, it turned out that the names of the songs stopped being displayed again. The attribute is with the value "stream_title: null"

the names are displayed in the MA itself, that is, the radio station transmits the names.

**Reply by OzGav** [reply]
OK I will transfer to an issue
